libs/openFrameworks/3d/ofEasyCam.cpp

```diff
@@ -12,23 +12,41 @@ static const unsigned long doubleclickTime = 200;
 //----------------------------------------
 ofEasyCam::ofEasyCam(){
 	reset();
+	sensitivityTranslate = {1,1,1};
+	sensitivityRot = {1,1,1};
+}
+//----------------------------------------
+void ofEasyCam::enableOrtho(){
+	if (!getOrtho()) {
+		prevFarClip = getFarClip();
+		prevNearClip = getNearClip();
+	}
+	setNearClip(-10000);
```

Comment:
I'm not sure if setting the far and near clip here as a side-effect can't get confusing - it doesn't seem to be what you ask for when you enable orthographic procection. 

Imagine what happens if someone thinks they know what they are doing and changes the near clip before enabling orthographic projection - they would find themselves surprised to be stuck with hard-coded -10000, 10000 which might be tricky to debug because it's a bit surprising =)